language/stdlib/modules/LBR.move

```diff
@@ -37,18 +38,24 @@ module LBR {
     // already be registered in order for this to succeed. The sender must
     // both be the correct address and have the correct permissions. These
     // restrictions are enforced in the Libra::register_currency function.
-    public fun initialize(association: &signer) {
-        assert(Signer::address_of(association) == 0xA550C18, 0);
+    public fun initialize(
+        association: &signer,
+        register_currency_capability: &Capability<RegisterNewCurrency>,
+        tc_capability: &Capability<TreasuryComplianceRole>,
```

Comment:
Naming question: should our convention be `tc_capability` or `tc_role` here?

Reply:
My thinking was `tc_capability` instead of `tc_role` here, since in my thinking the `TreasuryComplianceRole` is the "default" privilege for the TC role, and since it's wrapped in a capability this is a "capability" fully. But I don't have strong opinions about this so happy to rename if you have a preference for `tc_role` :)

Reply:
`tc_capability` makes sense to me; just wondering about your thinking.